functions/utils/getApolloClient.ts

```diff
@@ -0,0 +1,21 @@
+import { ApolloClient, InMemoryCache } from '@apollo/client'
+const GRAPHQL_ENDPOINT = 'https://api.uniswap.org/v1/graphql'
+
+export const getApolloClient = () => {
+  return new ApolloClient({
+    connectToDevTools: true,
+    uri: GRAPHQL_ENDPOINT,
+    headers: {
+      'Content-Type': 'application/json',
+      Origin: 'https://app.uniswap.org',
+      'User-Agent':
+        'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/114.0.5735.110 Safari/537.36',
+    },
+    cache: new InMemoryCache(),
```

Comment:
Do you know how caching will work within the worker? I assume it'd be hard to cache across requests.
Will endpoints with the same parameters just be cached for us automatically?